USERS.md

```diff
@@ -0,0 +1,6 @@
+Who uses Atlantis?
+As the Atlantis Community grows, we'd like to keep track of our users. Please send a PR with your organization name if you are using Atlantis.
+
+Currently, the following organizations are officially using Atlantis:
+
+1. Autodesk
```

Comment:
I think this is a great idea, but can we do a separate PR as well as an issue for this? :)